extensions/amp-video/0.1/amp-video.js

```diff
@@ -198,6 +198,10 @@ class AmpVideo extends AMP.BaseElement {
     this.createPosterForAndroidBug_();
     element.appendChild(this.video_);
 
+    // Fade out blur on poster load.
+    const posterEl = element.getElementsByTagName('i-amphtml-poster');
```

Comment:
This only exists on Android devices. `poster` is a built-in attribute on the `<video>` element. 

See if you can figure out how to detect poster loading on videos.